packages/react-next/src/components/Persona/PersonaCoin/PersonaCoin.test.tsx

```diff
@@ -70,12 +68,4 @@ describe('PersonaCoin', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
-
-  it('sets isImageLoaded to false if imageUrl changes from valid to undefined', () => {
```

Comment:
instead of removing the test, can you add a replacement? maybe something like - when imageUrl changes,`onPhotoLoadingStateChange` gets called with the correct value

Reply:
That would ultimately be testing `Image`, not `PersonaCoin`.